src/event-db/stage_data/prod/00001_fund12_event.sql

```diff
@@ -26,25 +26,25 @@ INSERT INTO event (
     extra,
     cast_to
 ) VALUES (
-    100,
-    'Fund 100',
-    'Catalyst Testnet - Fund 100',
-    '2024-01-15 21:45:00',  -- Registration Snapshot Time
-    '2024-01-15 22:00:00',  -- Snapshot Start.
+    12,
+    'Fund 12',
+    'Catalyst Testnet - Fund 12',
```

Comment:
Not sure if this is used somewhere, but since this is prod please remove `Testnet`.